### PR TITLE
dlgMovingAverages

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers - very minor changes but this is ready for review

### CODE REVIEW

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers This has been waiting for review for over a week

File: instat/dlgMovingAverage.vb [hunk 1]
```diff
@@ -1,30 +1,81 @@
...
+' (at your option) any later version.
+'
+' This program is distributed in the hope that it will be useful,
+' but WITHOUT ANY WARRANTY; without even the implied warranty of
+' MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License k
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.Imports instat
+Imports instat
```
Comment (stevenndungu) [inline instat/dlgMovingAverage.vb]: this line could be removed

File: instat/dlgMovingAverage.vb [hunk 2]
```diff
@@ -1,30 +1,81 @@
...
+
+        'chks
+        ucrChkDisplayResults.SetText("Display Results")
+        ucrChkLag.SetText("Lag")
+        ucrChkPlot.SetText("Plot")
+        ucrChkSaveResiduals.SetText("Save Residuals")
+
+        'ucrSave
+        ucrSaveResultsInto.SetCheckBoxText("Save Results into:")
+        ucrSaveResultsInto.SetIsComboBox()
```
Comment (stevenndungu) [inline instat/dlgMovingAverage.vb]: some lines missing, e.g setting the saveastype,

File: instat/dlgMovingAverage.vb [hunk 3]
```diff
@@ -1,30 +1,81 @@
...
+
+    Private Sub ucrBase_ClickReset(sender As Object, e As EventArgs) Handles ucrBase.ClickReset
+        SetDefaults()
+        SetRCodeForControls(True)
+        TestOKEnabled()
     End Sub
 
-    Private Sub ucrReceiverDataToAverage_Leave(sender As Object, e As EventArgs) Handles ucrReceiverDataToAverage.Leave
-        ucrBase.clsRsyntax.AddParameter("x", clsRFunctionParameter:=ucrReceiverDataToAverage.GetVariables())
+    Private Sub TestOKEnabled()
```
Comment (stevenndungu) [inline instat/dlgMovingAverage.vb]: I think the OK should be disabled now that the dialog is incomplete

**Comment by lilyclements** [comment]
@stevenndungu this dialog contains very minor changes because the dialog was not complete when I got to it. I was simply updating the controls and getting this dialog to run the same as it was before, and beforehand it was not running. Nothing is positioned because this dialog is not complete - I'm not sure what will be added in, moved around, etc so I felt it was best to not waste time on that and to instead update the controls and code so that it's functionality is the same as before.

File: instat/dlgMovingAverage.vb (hunk 2 above)
Comment (lilyclements) [inline instat/dlgMovingAverage.vb]: I have made it as functional as it was before, so since I don't know the savetype setting I have not done this yet.

**Comment by lilyclements** [comment]
@stevenndungu I have removed the Instat and made OK Enabled False, this is ready for re-review

**Comment by stevenndungu** [comment]
@lilyclements OK, i get you right. @dannyparsons  this is ready for you to review

**Pull request merged**